webpack.config.ts

```diff
@@ -42,6 +42,13 @@ function parseFeatureDetails(readmeContent: string, id: FeatureID): FeatureMeta
 		};
 	}
 
+	if (id.startsWith('rgh-')) {
+		return {
+			id,
+			description: '<p>Refined GitHub Meta feature</p>'
+		};
+	}
+
```

Comment:
This is happening too late and it leaves the feature in every list, with junk content.

Fixed in